R/fastMapGenerate.R

```diff
@@ -23,7 +23,7 @@
 #' @examples
 #' \dontrun{
 #' fastMapGenerate("hgnc_complete_set.txt", "symbol", "uniprot_ids",
-#'                 type = "UniProt", outName = "fastMapUniProt.rds")
+#'                 type = "UniProt", outputName = "fastMapUniProt.rds")
 #' }
```

Comment:
Another option is to raise an issue and assign it to him to check through the examples and make sure that they are consistent with the final functions.